libimage/load_test.go

```diff
@@ -80,3 +81,78 @@ func TestLoad(t *testing.T) {
 		}
 	}
 }
+
+func TestLoadReference(t *testing.T) {
+	// Make sure that loading images does not leave any artifacts in TMPDIR
+	// behind (containers/podman/issues/14287).
+	tmpdir := t.TempDir()
+	os.Setenv("TMPDIR", tmpdir)
```

Comment:
FYI you should use t.SetEnv (added in go 1.17) for new tests as it ensures that unsetenv is called (so no chance to forget that), anyhow no reason to block over that.